doc/overview.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Overview of dbxincluder
+=======================
+
+dbxincluder is an implementation of the XInclude 1.1 specification (https://www.w3.org/TR/xinclude-11)
+with support for DocBook transclusion (http://docbook.org/docs/transclusion).
+
+
+Limitations
+===========
+
+It does not support the XInclude 1.1 "xpointer" attribute, as the XPointer functionality is currently not exposed by lxml.
```

Comment:
Your statement isn't quite true either, while libxml2 support XPointer, lxml does not expose that functionality to python code.